azurerm/data_source_managed_disk.go

```diff
@@ -81,7 +91,13 @@ func dataSourceArmManagedDiskRead(d *schema.ResourceData, meta interface{}) erro
 
 	if props := resp.DiskProperties; props != nil {
 		if diskSize := props.DiskSizeGB; diskSize != nil {
-			d.Set("disk_size_gb", *diskSize)
+			d.Set("disk_size_gb", props.DiskSizeGB)
+		}
+		if diskIOPS := props.DiskIOPSReadWrite; diskIOPS != nil {
```

Comment:
We don't need the check here as d.Set() will do it for us:
```suggestion
					d.Set("disk_iops_read_write", props.DiskIOPSReadWrite)
```

Reply:
Fixed.